### ISSUE TITLE
At level 6, attacking a level 1 mob with a green difficulty bar gives 7 xp.  

### ISSUE THREAD

**Issue by GlitchyGirl** [project-rubika/issues#418]
**Describe the bug**
At level 6, attacking a level 1 mob with a green difficulty bar gives 7 xp.   The bar should either be 1) Grey, or 2) The mob should give more than 7xp (7x XP multiplier  * 1xp normally given for a grey mob)

**To Reproduce**
1. At level 6, attack a level 1 mob with a green hp bar.
2. Kill said mob
3. Observe XP gain

**Expected behavior**
The bar should either be 
1) Grey
or
2) The mob should give more than 7xp (7x XP multiplier  * 1xp normally given for a grey mob)

**Screenshots**
![image](https://github.com/project-rubika/issues/assets/43483963/4a764172-7920-4ff5-a025-e1c373b37527)

**Character info:**
NanoFae

```
Copy/paste the output of /getfull here
```
Nanofae [6 Fixer]
------------------------------------
    strength 	3
    stamina 	21
    agility 	21
    sense 	24
    psychic 	28
    intelligence 	33
    flags 	528961
    monsterscale 	100
    currentnano 	89
    headmesh 	40160
    cash 	701
    gmlevel 	511
    expansion 	3
    fatness 	0
    breed 	3
    visualbreed 	3
    profession 	4
    visualprofession 	4
    level 	6
    sex 	3
    visualsex 	3
    titlelevel 	1
    equippedweapontype 	20
    EquippedRHWeapon 	20
    health 	74
    currentncu 	0
    bodydevelopment 	34
    runspeed 	39
    submachinegun 	34
    distanceweaponinitiative 	35
    senseimprovement 	37
    psychologicalmodification 	38
    burst 	32
    nanoprowessinitiative 	30
    duck 	16
    dodge 	41
    evade 	16
    computerliteracy 	43
    breakingentry 	41
    exitstatel 	-1073740870
    externaldoorinstance 	-418378760
    externalplayfieldinstance 	3064
    selectedtarget 	717
    selectedtargettype 	0
    isfightingme 	0
    charstate 	0
    xp 	20317
    currentmovementmode 	3
    waitstate 	1
    nextxp 	20650
    lastsavexp 	4500
    firstaid 	20
    treatment 	41
    lastxp 	15650
    materiallocation 	38
    materialcreation 	9
    biologicalmetamorphose 	39
    materialmetamorphose 	39
    playfieldtype 	0
    maxncu 	8
    monsterdata 	0
    team 	0
    side 	0
    race 	1
    accountflags 	1234567890
    life 	108
    VisualFlags 	31
    clan 	0
    healinterval 	29
    maxnanoenergy 	89
    nanoenergypool 	11
    nanointerval 	28
    aggdef 	100
    state 	0
    unarmedtemplateinstance 	0
    invaderskilled 	0
    killedbyinvaders 	0
    vp 	0
    unsavedxp 	0
    nanofocuslevel 	0
    specialization 	0
    shadowbreedtemplate 	0
    shadowbreed 	0
    lastperkresettime 	0
    socialstatus 	0
    playeroptions 	0
    tempsaveteamid 	0
    tempsaveplayfield 	0
    tempsavex 	0
    tempsavey 	0
    PVPDuelKills 	0
    PVPDuelDeaths 	0
    PVPProfessionDuelKills 	0
    PVPProfessionDuelDeaths 	0
    PVPRankedSoloKills 	0
    PVPRankedSoloDeaths 	0
    PVPRankedTeamKills 	0
    PVPRankedTeamDeaths 	0
    PVPSoloScore 	0
    PVPTeamScore 	0
    PVPDuelScore 	0
    unreadmailcount 	0
    lastmailchecktime 	1283065897
    savedxp 	0
    features 	6
    apartmentsallowed 	1
    apartmentsowned 	0
    nanoac 	5
    lastconcreteplayfieldinstance 	0
    mapoptions 	0
    mapareapart1 	0
    mapareapart2 	0
    mapareapart3 	0
    mapareapart4 	0
    missionbits1 	0
    missionbits2 	0
    missionbits3 	0
    missionbits4 	0
    missionbits5 	0
    missionbits6 	0
    missionbits7 	0
    missionbits8 	0
    missionbits9 	0
    missionbits10 	0
    missionbits11 	0
    missionbits12 	0
    sessiontime 	1234567890
    autoattackflags 	5
    personalresearchlevel 	0
    globalresearchlevel 	0
    personalresearchgoal 	0
    globalresearchgoal 	0
    battlestationside 	0
    battlestationrep 	10
    members 	999
    veteranpoints 	0
    monthspaid 	0
    PaidPoints 	0
    xpkillrange 	5
    inplay 	0
    attitude 	0
    alignment 	0
    icon 	0
    mesh 	17530
    deadtimer 	0
    martialarts 	9
    meleemultiple 	8
    onehandbluntweapons 	8
    onehandedgedweapon 	8
    piercing 	9
    twohandbluntweapons 	8
    twohandedgedweapons 	7
    meleeenergyweapon 	11
    throwingknife 	9
    grenade 	11
    throwngrapplingweapons 	8
    bow 	9
    pistol 	10
    rifle 	10
    shotgun 	8
    assaultrifle 	9
    drivewater 	11
    closecombatinitiative 	11
    physicalprowessinitiative 	11
    bowspecialattack 	10
    mechanicalengineering 	11
    electricalengineering 	11
    lr_energyweapon 	11
    lr_multipleweapon 	11
    disarmtrap 	11
    perception 	11
    adventuring 	9
    swim 	9
    driveair 	11
    mapnavigation 	12
    tutoring 	12
    brawl 	7
    riposte 	10
    dimach 	11
    parry 	8
    sneakattack 	11
    fastattack 	10
    flingshot 	10
    aimedshot 	11
    fieldquantumphysics 	12
    weaponsmithing 	9
    pharmaceuticals 	12
    nanoprogramming 	13
    psychology 	12
    chemistry 	11
    concealment 	10
    driveground 	11
    fullauto 	7
    currentmass 	0
    itemtype 	0
    previoushealth 	50
    currentstate 	0
    age 	0
    radiationac 	0
    attackspeed 	5
    metatype 	0
    fireac 	0
    poisonac 	0
    coldac 	0
    chemicalac 	0
    energyac 	0
    meleeac 	0
    projectileac 	0
    rp 	0
    specialcondition 	1
    sk 	0
    clanredeemed 	0
    clanconserver 	0
    clandevoted 	0
    otunredeemed 	0
    otoperator 	0
    otfollowers 	0
    gos 	0
    clanvanguards 	0
    ottrans 	0
    clangaia 	0
    otmed 	1234567890
    clansentinels 	0
    otarmedforces 	0
    playerid 	1234567890
    alienlevel 	0
    aliennextxp 	1500
    alienxp 	0
    insurancepercentage 	0
    prevmovementmode 	3
    teamside 	0
    beltslots 	0
    absorbprojectileac 	0
    absorbmeleeac 	0
    absorbenergyac 	0
    absorbchemicalac 	0
    absorbradiationac 	0
    absorbcoldac 	0
    absorbnanoac 	0
    absorbfireac 	0
    absorbpoisonac 	0
    temporaryskillreduction 	0
    insurancetime 	0
    mapflags 	0
    changesidecount 	0
------------------------------------

**Playfield info:**

```
Copy/paste the output of Shift+F9 here
```
------------------------------------

Extended location information:
- 123.4, 80.6, 4.4 (123.4 80.6 y 4.4 954)
- Pf Proxy: Model=51100:954 GS=0 SG=0 R=954, resource: 954
- zone: 19, area: "Junkyard"
Server id: 1, character id: 50000:717, time: 2024-01-11 17:30:50 (UTC)
Version: 00.4.2_EP1, map id: NOT FOUND, build: 52606.

----

**Comment by TheTinkeringIdiot** [comment]
Does not replicate exactly as described, but this is an issue. At level 6, level 1 mobs display a green bar but only award 1xp each upon death. 1xp should be reserved for grey mobs only, greens should award more.

**Comment by RaggyLP** [comment]
This specific issue is now fixed as part of #795.
https://github.com/project-rubika/issues/issues/795#issuecomment-2310374169
Closing as fixed as of 31/08/2024.
